head     1.1;
branch   1.1.1;
access   ;
symbols  LLVM_5_0_0:1.1.1.1 LLVM:1.1.1;
locks    ; strict;
comment  @# @;
expand   @o@;


1.1
date     2017.10.04.20.27.42;  author patrick;  state Exp;
branches 1.1.1.1;
next     ;
commitid        ufzi3t8MqoilCPqO;

1.1.1.1
date     2017.10.04.20.27.42;  author patrick;  state Exp;
branches ;
next     ;
commitid        ufzi3t8MqoilCPqO;


desc
@@



1.1
log
@Initial revision
@
text
@"""Tests that an exceptional exit is detected by the testbot."""

from __future__ import print_function

import atexit
import os
import signal
import time

import lldbsuite.test.lldbtest as lldbtest


class ExceptionalExitOutOfTestMethodTestCase(lldbtest.TestBase):
    """Forces exceptional exit."""
    mydir = lldbtest.TestBase.compute_mydir(__file__)

    @@lldbtest.skipIfWindows
    def test_buildbot_catches_exceptional_exit(self):
        pass

def cleanup():
        os.kill(os.getpid(), signal.SIGKILL)

atexit.register(cleanup)
@


1.1.1.1
log
@Import LLVM 5.0.0 release including clang, lld and lldb.
@
text
@@
